admin-console/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.entando</groupId>
         <artifactId>app-engine</artifactId>
-        <version>7.1.4</version>
+        <version>7.1.5-SNAPSHOT</version>
```

Comment:
@BrenoQVDS can you change to 7.1.5? *-SNAPSSHOT only for develop

Reply:
Sure, done!